src/QtUtils/MockCliProgram/main.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/strings/numbers.h>
+
+#include <chrono>
+#include <iostream>
+#include <string_view>
+#include <thread>
+
+int main(int argc, char* argv[]) {
+  if (argc < 2) {
+    std::cout << "Some example output" << std::endl;
+    return 0;
+  }
+
+  if (std::string_view{argv[1]} == "--sleep_for_ms") {
+    int timeout = 0;
+    bool convert_result = absl::SimpleAtoi(argv[2], &timeout);
+    if (!convert_result) {
+      std::cerr << "Wrong format of arguments" << std::endl;
+      return 1;
+    }
+    std::this_thread::sleep_for(std::chrono::milliseconds{timeout});
+    std::cout << "Slept for " << timeout << "ms" << std::endl;
+    return 0;
+  }
+
+  if (std::string_view{argv[1]} == "--exit_code") {
+    int result = 0;
+    bool convert_result = absl::SimpleAtoi(argv[2], &result);
+    if (!convert_result) {
+      std::cerr << "Wrong format of arguments" << std::endl;
+      return 1;
+    }
+    return result;
+  }
+}
```

Comment:
Again, I know it's just for testing, but handle the case of `argc>=2` but unexpected options?

Reply:
I changed it to use absl flags